lib/interfaces.ts

```diff
@@ -61,6 +61,7 @@ export interface FireFlyWebSocketOptions {
   autoack: boolean;
   reconnectDelay: number;
   heartbeatInterval: number;
+  headers?: Record<string, string>;
```

Comment:
Rather than just exposing `headers`, should we go as far as to expose _all_ websocket options for overriding (similar to how we expose all `AxiosRequestConfig` on the HTTP requests)? ie should we just have `options: WebSocket.ClientOptions | ClientRequestArgs` here?

Reply:
Good shout - I'll update the PR :+1: